client/pubspec.yaml

```diff
@@ -59,8 +64,9 @@ flutter:
   uses-material-design: true
 
   # To add assets to your application, add an assets section, like this:
-  # assets:
-  #   - images/a_dot_burr.jpeg
+  assets:
+  -  C:\Users\devru\Digital-Entry-System\client\lib\Assets\iit_logo_original.png
+  #Yaar relative path kaam hi nhi kar rha tha mere laptop me isliye maine absolute path daal diya.
```

Comment:
`- lib/Assets/iit_logo_original.png`
does this not work? it seems to work for me

Reply:
Mene to try Kara tha, mereme kaam nhi kra, doesn't exist aa rha tha😭😭

Reply:
> `- lib/Assets/iit_logo_original.png` does this not work? it seems to work for me

this works for me too.